dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/.template.config/localize/templatestrings.cs.json

```diff
@@ -1,7 +1,7 @@
-{
-  "author": "Microsoft",
-  "name": "MacCatalyst Application",
-  "description": "Projekt pro vytvoření aplikace .NET MacCatalyst",
-  "symbols/bundleId/description": "Přepíše CFBundleIdentifier v souboru Info.plist.",
-  "symbols/minOSVersion/description": "Přepíše SupportedOSPlatformVersion v souboru projektu."
+{
+  "author": "Microsoft",
+  "name": "Mac Catalyst Application",
+  "description": "A project for creating a .NET Mac Catalyst application",

```

Comment:
@tj-devel709 this looks like it's reverting text back to English?

Reply:
It looks like this is happening since the "name" was changed (a space was added between MacCatalyst). I think it is safe to assume that the text will be what we want in the next iteration

Reply:
Actual I will consult with the Loc team to confirm

Reply:
Okay yes. 2 months ago, the description was "A project for creating a .NET MacCatalyst application" but now there is a space in between "Mac" and "Catalyst" in the description hence the change